pages/components/feed.js

```diff
@@ -0,0 +1,11 @@
+import Cards from "./cards";
+import ActiveMusic from "./activeMusic";
+
+export default function Feed() {
+  return (
+    <div>
+      <ActiveMusic />
+      <Cards />
+    </div>
```

Comment:
Did you know that you could use [Fragments](https://react.dev/reference/react/Fragment#returning-multiple-elements) to return multiple elements instead of a div?


By the way: 
I would suggest you not to nest the components `<ActiveMusic />` and `<Cards />` , because they are very small anyway.  Rather import/use them in `page/index.js` - **but that's more a suggestion** ✨

```suggestion
    <>
      <ActiveMusic />
      <Cards />
    </>
```

Reply:
change it. Thanks